app/mailers/rad_mailer.rb

```diff
@@ -39,11 +39,15 @@ def simple_message(recipient, subject, message, options = {})
 
     @message = options[:do_not_format] ? message : simple_format(message)
     @email_action = options[:email_action] if options[:email_action]
-    enable_settings_link if options[:notification_settings_link]
 
     maybe_attach options
 
-    mail(to: to_address, subject: subject, cc: options[:cc], bcc: options[:bcc])
+    mail to: to_address,
+         subject: subject,
+         cc: options[:cc],
+         bcc: options[:bcc],
+         template_path: 'rad_mailer',
```

Comment:
adding the template path and name was required now that more than one mailer are using this template